content/ja/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -1,218 +1,237 @@
+`label-1:key-1`ラベル、もう一つは`label-2:key-2`ラベルで、スケジューラーは各Nodeの`weight`を考察し、その重みとNodeの他のスコアを加え、最終スコアが最も高いNodeにPodをスケジューリングします。
 
-`nodeSelectorTerms`内で複数の`matchExpressions`を指定した場合にはPodは**全ての**`matchExpressions`を満たしたNodeへスケジュールされます。
+{{<note>}}
+この例でKubernetesにPodを正常にスケジュールさせるには、`kubernetes.io/os=linux`ラベルを持っているNodeの存在を確保しないといけません。
+{{</note>}}
 
-PodがスケジュールされたNodeのラベルを削除したり変更しても、Podは削除されません。
-言い換えると、アフィニティはPodをスケジュールする際にのみ考慮されます。
+#### スケジューリングプロファイルごとのNodeアフィニティ {#node-affinity-per-scheduling-profile}
 
-`preferredDuringSchedulingIgnoredDuringExecution`内の`weight`フィールドは、1から100の範囲で指定します。
-全ての必要条件(リソースやRequiredDuringSchedulingアフィニティ等)を満たしたNodeに対して、スケジューラーはそのNodeがMatchExpressionsを満たした場合に、このフィルードの"weight"を加算して合計を計算します。
-このスコアがNodeの他の優先機能のスコアと組み合わせれ、最も高いスコアを有したNodeが優先されます。
+{{< feature-state for_k8s_version="v1.20" state="beta" >}}
 
-### Pod間アフィニティとアンチアフィニティ
+複数の[スケジューリングプロファイル](/ja/docs/reference/scheduling/config/#multiple-profiles)を設定する場合、プロファイルにNodeアフィニティを関連付けることができます。これは、プロファイルが特定のNode群にのみ適用される場合に便利です。[スケジューラーの設定](/ja/docs/reference/scheduling/config/)にある[`NodeAffinity`プラグイン](/ja/docs/reference/scheduling/config/#scheduling-plugins)の`args`フィールドに`addedAffinity`を追加すると実現できます。例えば:
 
-Pod間アフィニティとアンチアフィニティは、Nodeのラベルではなく、すでにNodeで稼働しているPodのラベルに従ってPodがスケジュールされるNodeを制限します。
-このポリシーは、"XにてルールYを満たすPodがすでに稼働している場合、このPodもXで稼働させる(アンチアフィニティの場合は稼働させない)"という形式です。
-Yはnamespaceのリストで指定したLabelSelectorで表されます。
-Nodeと異なり、Podはnamespaceで区切られているため(それゆえPodのラベルも暗黙的にnamespaceで区切られます)、Podのラベルを指定するlabel selectorは、どのnamespaceにselectorを適用するかを指定する必要があります。
-概念的に、XはNodeや、ラック、クラウドプロバイダゾーン、クラウドプロバイダのリージョン等を表すトポロジードメインです。
-これらを表すためにシステムが使用するNodeラベルのキーである`topologyKey`を使うことで、トポロジードメインを指定することができます。
-先述のセクション[補足: ビルトインNodeラベル](#interlude-built-in-node-labels)にてラベルの例が紹介されています。
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+
+profiles:
+  - schedulerName: default-scheduler
+  - schedulerName: foo-scheduler
+    pluginConfig:
+      - name: NodeAffinity
+        args:
+          addedAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: scheduler-profile
+                  operator: In
+                  values:
+                  - foo
+```
 
+`addedAffinity`は、Podの仕様(spec)で指定されたNodeアフィニティに加え、`.spec.schedulerName`を`foo-scheduler`に設定したすべてのPodに適用されます。つまり、Podにマッチするためには、Nodeは`addedAffinity`とPodの`.spec.NodeAffinity`を満たす必要があるのです。
+
+`addedAffinity`はエンドユーザーには見えないので、その動作はエンドユーザーにとって予期しないものになる可能性があります。スケジューラープロファイル名と明確な相関関係のあるNodeラベルを使用すべきです。
 
 {{< note >}}
-Pod間アフィニティとアンチアフィニティは、大規模なクラスター上で使用する際にスケジューリングを非常に遅くする恐れのある多くの処理を要します。
-そのため、数百台以上のNodeから成るクラスターでは使用することを推奨されません。
+[DaemonSetのPodを作成する](/ja/docs/concepts/workloads/controllers/daemonset/#デフォルトスケジューラーによってスケジューリングされる場合)DaemonSetコントローラーは、スケジューリングプロファイルをサポートしていません。DaemonSetコントローラーがPodを作成すると、デフォルトのKubernetesスケジューラーがそれらのPodを配置し、DaemonSetコントローラーの`nodeAffinity`ルールに優先して従います。
```

Comment:
Can you add an English ID for the target page and change to use the English ID instead?

Reply:
Thank you for your kind advice! I've changed to use the English ID.
And I noticed that [the original English version daemonset page ](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#how-daemon-pods-are-scheduled) has been updated lately, and the ID used here is deleted. 
Therefore the ID I fixed here will be consistent with [the current Japanese translation of daemonset page](https://kubernetes.io/ja/docs/concepts/workloads/controllers/daemonset/#%E3%83%87%E3%83%95%E3%82%A9%E3%83%AB%E3%83%88%E3%82%B9%E3%82%B1%E3%82%B8%E3%83%A5%E3%83%BC%E3%83%A9%E3%83%BC%E3%81%AB%E3%82%88%E3%81%A3%E3%81%A6%E3%82%B9%E3%82%B1%E3%82%B8%E3%83%A5%E3%83%BC%E3%83%AA%E3%83%B3%E3%82%B0%E3%81%95%E3%82%8C%E3%82%8B%E5%A0%B4%E5%90%88), but not with the English version. 
I'll create an Issue about update the daemonset page, and will start working on it as soon as possible.

Reply:
Attached the Issue link here just in case. https://github.com/kubernetes/website/issues/39502

Reply:
@nasa9084 @t-inu @atoato88 About the ID here, the [original English page](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity-per-scheduling-profile)  (The Note part, sorry you may need to scroll down a bit.) is still using `#scheduled-by-default-scheduler` ,  but `#scheduled-by-default-scheduler`part in [DeamonSet page](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#scheduled-by-default-scheduler) has [been deleted](https://github.com/kubernetes/website/commit/d0b3ba5ccea5138c06ece1ad9f4e4c2b0c5ef897#), so I changed to `#how-daemon-pods-are-scheduled` in the Japanese version.
Could I ask if you think this modification is appropriate or not? 
Looking forward to your comments and many thanks in advance!